google/resource_compute_target_ssl_proxy_test.go

```diff
@@ -103,6 +104,19 @@ func testAccCheckComputeTargetSslProxyExists(n string) resource.TestCheckFunc {
 			return fmt.Errorf("TargetSslProxy not found")
 		}
 
+		if found.ProxyHeader != proxyHeader {
+			return fmt.Errorf("Wrong proxy header. Expected '%s', got '%s'", proxyHeader, found.ProxyHeader)
+		}
+
+		foundCertsName := make([]string, 0, len(found.SslCertificates))
+		for _, foundCert := range found.SslCertificates {
+			foundCertsName = append(foundCertsName, GetResourceNameFromSelfLink(foundCert))
+		}
+
+		if !reflect.DeepEqual(foundCertsName, sslCerts) {
```

Comment:
Does order matter? If it doesn't, you might consider sorting or something before doing the deepEqual

Reply:
Actually, after waiting for the full test run to finish and reading the API docs, one and only one certificate must be specified at this time (they might support more in the future). So I added a `MaxItems:1` on the schema for now and simplified this check.

From the docs:
"Currently exactly one SslCertificate resource must be specified."